http/restful-ws-jakarta/pom.xml

```diff
@@ -0,0 +1,96 @@
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.cloudevents</groupId>
+        <artifactId>cloudevents-parent</artifactId>
+        <version>2.4.0-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>cloudevents-http-restful-ws-ee9</artifactId>
+    <name>CloudEvents - Jakarta EE9 RESTful Web Services Http Binding</name>
+    <packaging>jar</packaging>
+
+    <pluginRepositories>
+        <pluginRepository>
+            <id>ossrh</id>
+            <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+            <layout>default</layout>
+            <releases>
+                <enabled>true</enabled>
+            </releases>
+        </pluginRepository>
+    </pluginRepositories>
+
+    <properties>
+        <jakarta-ee.version>3.0.0</jakarta-ee.version>
+        <jersey.version>3.0.8</jersey.version>
+        <vertx.version>3.9.0</vertx.version>
+        <resteasy.version>4.5.3.Final</resteasy.version>
+        <module-name>io.cloudevents.http.restfulws</module-name>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.cloudevents</groupId>
+            <artifactId>cloudevents-core</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>jakarta.ws.rs</groupId>
+            <artifactId>jakarta.ws.rs-api</artifactId>
+            <version>${jakarta-ee.version}</version>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <artifactId>maven-antrun-plugin</artifactId>
+                <version>3.0.0</version>
+                <executions>
+                    <execution>
+                        <phase>generate-sources
+                        </phase>
+                        <configuration>
+                            <target>
+                                <delete dir="src/main"/>
+                                <copy todir="src">
+                                    <fileset dir="../restful-ws\src" />
+                                    <filterchain>
+                                        <replaceregex pattern="javax." replace="jakarta." />
+                                    </filterchain>
+                                </copy>
+                                <!--<replace dir="src/main/java/io/cloudevents/http/restful/ws" token="javax" value="jakarta">
+                                    <include name=" **/*.java" />
+                                </replace>-->
```

Comment:
Can we remove this commented code?

Reply:
@pierDipi I am working on a new Integration test case at the moment and as part of that commit I will address both items

Reply:
@pierDipi I have added the new test case, found a few other items that needed cleaning up and adding documentation for the new package. so I consider this ready for a deeper review